### ISSUE TITLE
React logo link not accessible

### ISSUE THREAD

**Issue by xydlebug** [xydlebug/X-ttt#3]
The "home page" link wrapping the React logo does not have any text description.

See https://dequeuniversity.com/rules/axe/4.8/link-name?application=AxeChrome
